### PR TITLE
Setup python development server

### PR DESCRIPTION
**Pull request by fungjj92**
## Overview

As we migrate the app to a static site, we'll need our own dev environment apart from the current Visual Studio .NET setup from which we can work and model static site architecture. As per the ADR, we will use a python environment and thusly for the dev server use python's SimpleHTTPServer. Keeping to wider Azavea convention, we will use the STRTA workflow.

Connects #1085 

### Demo
screen shot 2018-09-11 at 4 38 27 pm

### Notes

There's one more 403 accessing `use.js` -- the file path is incorrect, as I tried to solve for other similar failures in [`e56043e`](https://github.com/CoastalResilienceNetwork/GeositeFramework/commit/e56043eb29798110154b44870865f6720ffcf138), but I wasn't able to figure out where `use.js` was even being called and loaded. It doesn't seem like a big deal and will likely be fixed as we migrate and update the rest of the app and assets, which will additionally address the rest of the expected console errors.

## Testing Instructions

Either: 
- Clone this repo locally, *not in your windows VM*, and checkout the branch
or
- Pull down this branch in the Windows VM: *do not* run the project in Visual Studio

Then:
- `./scripts/server`
- open a working index page (with a lot of broken templated code, intentionally) at http://localhost:54633

### CODE REVIEW

**Comment by caseycesari** [comment]
Use.js is located here:

> `src/GeositeFramework/js/lib/use.js`

**Comment by caseycesari** [comment]
Oh sorry. Misread the comment. Yeah, I think use.js loading depends on some [back-end processing](https://github.com/CoastalResilienceNetwork/GeositeFramework/blob/128f17cb97401d41ee359f33cb06ae99b6804161/src/GeositeFramework/Models/Geosite.cs#L339). Want to capture that in an issue so we don't forget?

**Comment by fungjj92** [comment]
screen shot 2018-09-12 at 4 23 11 pm

Note, there are a few new console errors for improper pathnames for font files. Adding in issue #1108

File: scripts/server [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    def translate_path(self, path):
+        words = filter(None, path.split('/'))
+        modified_path = self.base_path
+        for word in words:
+            modified_path = os.path.join(modified_path, word)
+        return modified_path
+
+def serve():
+    server = CustomRootHTTPServer(PATH_TO_PROJECT, ('', PORT), CustomRootHTTPRequestHandler)
+    server.serve_forever()
```
Comment (caseycesari) [inline scripts/server]: Minor, but consider logging a message that the server has started.

Comment (fungjj92) [reply scripts/server]: I like it

**Comment by fungjj92** [comment]
Logging added. Merging

screen shot 2018-09-13 at 3 51 42 pm

File: scripts/server [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+PATH_TO_PROJECT = 'src/GeositeFramework/'
+
+logging.basicConfig(level=logging.INFO)
+
+class CustomRootHTTPServer(HTTPServer):
+    def __init__(self, base_path, *args, **kwargs):
+        HTTPServer.__init__(self, *args, **kwargs)
+        self.RequestHandlerClass.base_path = base_path
+
+class CustomRootHTTPRequestHandler(SimpleHTTPRequestHandler):
```
Comment (fungjj92) [inline scripts/server]: http://louistiao.me/posts/python-simplehttpserver-recipe-serve-specific-directory/

**Pull request merged**